tests/Integration/PrestaShopBundle/Controller/Admin/Improve/Design/PositionsControllerTest.php

```diff
@@ -26,14 +26,38 @@
 
 namespace Tests\Integration\PrestaShopBundle\Controller\Admin\Improve\Design;
 
+use Cache;
+use Hook;
+use Module;
 use Symfony\Component\HttpFoundation\Response;
 use Tests\Integration\PrestaShopBundle\Test\WebTestCase;
+use PrestaShop\PrestaShop\Core\Addon\Module\ModuleManager;
 
 /**
  * @group demo
  */
 class PositionsControllerTest extends WebTestCase
 {
+    protected $moduleId;
+    protected $hookId;
+
+    public function setUp()
+    {
+        Cache::clear();
+        Module::$id2name = null;
```

Comment:
This is a strange artifact :(
I'm afraid we will forget what it is, what it means and how to use it if we do not wrap it in some function like `Module::clearStupidStaticCache()`

Reply:
Done! Thanks.